web/crux-ui/src/components/products/versions/images/config/crane-config-section.tsx

```diff
@@ -35,9 +35,15 @@ const CraneConfigSection = (props: CraneConfigSectionProps) => {
     propsOnChange(newConfig)
   }
 
-  const contains = (filter: ImageConfigFilterType): boolean => filters.indexOf(filter) !== -1
+  const contains = (filter: CraneConfigFilterType, checkAny: boolean = false): boolean => {
+    if (checkAny) {
```

Comment:
Personally i dont like mixing these kind of checks. Please extract it to another function. I think you make this function to be as common as possible and move it to the images.ts.

Reply:
refactored thx